tests/.test_pylintrc

```diff
@@ -0,0 +1,35 @@
+# The idea is to use this file and remove disabled rule until we're able to merge with the pylintrc used for pylint
```

Comment:
Good idea.

The next step could be to add `# pylint disable=<problem-this-file>` to each file. That way each new file added, and existing files that do not exhibit `<problem-this-file>`, will not regress.

Reply:
👍 I'm going to do that after fixing the easier one, in #3496.

Reply:
I'm actually way ahead of you on that. :) PR in-bound.

Reply:
I did not touch the redefined-outer-name, but fixed pretty much everything else.

Reply:
Oh, you've fixed them, because I'm risk averse I just added `disable=` to problem files, but what you have is even better, and would be better to get in (obvs). :)

Reply:
You can rebase on the latest master to get it, I think it's non-controversial changes so I merged it already. I was half-afraid that we did something incompatible yesterday by reading your previous comment.. Good thing we did not :)